sdk/cosmos/azure-spring-data-cosmos/src/samples/java/com/azure/cosmos/multidatasource/DatabaseConfiguration.java

```diff
@@ -2,14 +2,22 @@
 // Licensed under the MIT License.
 package com.azure.cosmos.multidatasource;
 
+import com.azure.cosmos.CosmosAsyncClient;
 import com.azure.cosmos.CosmosClientBuilder;
 import com.azure.cosmos.CosmosProperties;
 import com.azure.spring.data.cosmos.CosmosFactory;
 import com.azure.spring.data.cosmos.config.AbstractCosmosConfiguration;
+import com.azure.spring.data.cosmos.config.CosmosClientConfig;
 import com.azure.spring.data.cosmos.config.CosmosConfig;
+import com.azure.spring.data.cosmos.core.CosmosTemplate;
 import com.azure.spring.data.cosmos.core.ReactiveCosmosTemplate;
+import com.azure.spring.data.cosmos.core.ResponseDiagnostics;
+import com.azure.spring.data.cosmos.core.ResponseDiagnosticsProcessor;
 import com.azure.spring.data.cosmos.core.convert.MappingCosmosConverter;
 import com.azure.spring.data.cosmos.repository.config.EnableReactiveCosmosRepositories;
+import io.micrometer.core.lang.Nullable;
```

Comment:
We should get this import from springframework
Please use this and everywhere else -> `import org.springframework.lang.Nullable;`

Spring should not take a dependency on `micrometer` module as it is not required.

Reply:
There is also one more place where this is used -> `AppConfiguration.java` - please update that one as well.